edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+ARG DIR=.
+
+FROM alpine:3.12.0
+WORKDIR /app
+COPY target/x86_64-unknown-linux-musl/release/api-proxy-module .
+COPY templates .
+RUN ls
```

Comment:
nit: still required?

Reply:
Yes, this is used by the script to generate the image. I have mimicked this: https://github.com/Azure/iotedge/blob/master/edge-modules/edgehub-proxy/build.sh

Reply:
Done!